### PR TITLE
Bump black to > 22

### PR DESCRIPTION
**Pull request by sstrasser-globality**
In trying to get some of the AI services working on a new M1 Apple PC, we ran into an issue with some of the linting tools. Messing around to see if this fixes the problem.

### CODE REVIEW

**Review by marcglobality — COMMENTED** [review]
I'm ok with the changes but before approving I would like @uri-miron-globality to review this as well as he brought the tomli pin some PRs ago

File: setup.py [hunk 1]
```diff
@@ -36,21 +37,20 @@
...
     install_requires=[
-        "Click>=7.0,<8.0",
-        # TODO: remove tomli pinning when black version is changed to 22.x
-        "tomli<2.0.0",
+        "Click>=8.0",
+        "tomli>=2.0.0",
         "parso",
         "pytest>=3",
-        "black==21.7b0",
+        "black>=22.1.0",
```
[the remark this answers is not in the document]
Comment (sstrasser-globality) [reply setup.py:44]: Woops. Yah, good call!

[the remark this answers is not in the document]
Comment (uri-miron-globality) [reply setup.py:44]: Yes, I investigated this awhile back. The issue is that marquez is pinning flask<2, which pins click<8, which conflicts with black>=22. The solution is for marquez to update requirements... Until this is done, the only way I could build a repo that uses both globality-black and marquez was to pin both black and click in this file.

**Review by uri-miron-globality — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcglobality) [reply setup.py:43]: > please leave pinned. If you want, to 22.1.0. But we don't want black to suddenly change into something that makes all our CIs to start failing

I'm going to contradict myself and say that if better to break CIs than being stuck in old requirements. So as long as we fix the issues brought up by @uri-miron-globality , let's do this

[the remark this answers is not in the document]
Comment (sstrasser-globality) [reply setup.py:43]: I think you already fixed this issue here: https://github.com/globality-corp/marquez/pull/3814/files? So we should be able to roll this forward with almost minimal impact.

**Review by tomaszpietruszka-globality — APPROVED** [review]
LGTM, as long as we can test if with repos of different types (AI, backend service) and it works - lets unpin!

**Review by uri-miron-globality — APPROVED** [review]
LGTM

**Pull request merged**